integration/setup/tenant_cluster.go

```diff
@@ -0,0 +1,135 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/integration/env"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
+	"github.com/giantswarm/microerror"
+)
+
+func CreateTenantCluster(ctx context.Context, config Config) error {
+	vpcID, err := ensureHostVPC(ctx, config)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = installAWSConfig(ctx, config, vpcID)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func DeleteTenantCluster(ctx context.Context, config Config) error {
+	err := config.Release.Delete(ctx, "apiextensions-aws-config-e2e")
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func ensureHostVPC(ctx context.Context, config Config) (string, error) {
+	stackName := "host-peer-" + env.ClusterID()
+
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating stack %#q", stackName))
+
+		os.Setenv("AWS_ROUTE_TABLE_0", env.AWSRouteTable0())
+		os.Setenv("AWS_ROUTE_TABLE_1", env.AWSRouteTable1())
+		os.Setenv("CLUSTER_NAME", env.ClusterID())
```

Comment:
This still needs improvements. `e2etemplates.AWSHostVPCStack` should be moved to `e2etemplates` repo.